src/app/qgsmaptoolfeatureaction.cpp

```diff
@@ -96,6 +95,7 @@ bool QgsMapToolFeatureAction::doAction( QgsVectorLayer *layer, int x, int y )
     return false;
 
   QgsPointXY point = mCanvas->getCoordinateTransform()->toMapCoordinates( x, y );
+  QPoint position = mCanvas->mapToGlobal( QPoint( x + 5, y + 5 ) );
```

Comment:
Where does the +5 come from?

Reply:
I took it from here https://github.com/qgis/QGIS/blob/master/src/gui/qgsmaptoolidentify.cpp#L123
I thought it's good to act the same. Although I think few people would notice when it's different...